app/models/post.rb

```diff
@@ -2,6 +2,9 @@ class Post < ApplicationRecord
   belongs_to :author, class_name: 'User'
   has_many :comments
   has_many :likes
+  validates :Title, presence: true, length: { maximum: 250 }
```

Comment:
- [ ] Good job adding the validation, but the test you wrote for title validation is not passing. In your test, you check if the title has 250 characters, which is the case, but you expect that to be invalid while it's a valid `" subject.Title = 'a' * 250"`It's correct because the title has 250 characters; it can be invalid if the title exceeds 250 likes 251. Please kindly fix that 🙃 

<img width="1126" alt="Screenshot 2023-12-08 at 19 29 30" src="https://github.com/ismayito/Rails-blog-app/assets/74501523/48edd91e-926b-4b35-967d-7e04e8559eaa">